### PR TITLE
Placeholder Home Screen

### PR DESCRIPTION
**Pull request by ajshort**
I also extracted some repeated styles into a separate file.

### CODE REVIEW

File: client/src/screens/home.screen.js [hunk 1]
```diff
@@ -1,108 +1,160 @@
...
+  eventUrgent: {
+    borderColor: 'red'
+  }
 });
 
 class Home extends Component {
   static navigationOptions = {
     title: 'Home',
-    tabBarIcon: ({ tintColor}) => <Icon size={30} name={'home'} color={tintColor} />
+    tabBarIcon: ({ tint }) => <Icon size={48} name='home' color={tint} />
```
Comment (sdunster) [inline client/src/screens/home.screen.js]: reverting this size increase cause it looks dumb on iOS. Clearly we just need to handle the styles differently on iOS vs android but I have more important things to work on right now. Feel free to take it on if you'd like.

**Comment by sdunster** [comment]
Landed in b0723b1

File: client/src/screens/home.screen.js (hunk 1 above)
Comment (ajshort) [reply client/src/screens/home.screen.js]: Good spot, the size change wasn't deliberate - I was probably trying to fiddle with the power icon size.

Comment (sdunster) [reply client/src/screens/home.screen.js]: The variable name change here actually broke the tint colour being applied as well. This is a JS unpacking construct so the name needs to match what is in the argument passed to this func. There's ways to get around this but you can google.

**Pull request closed without merging**